pytest-selenium/utils/utility.py

```diff
@@ -213,238 +213,285 @@ class Library(object):
-            "search_results_total": 18,
+            "chapter_search_results_total": 18,
+            "rkt_search_results_total": 1,
         },
         "principles-macroeconomics-ap-courses-2e": {
             "default_page": "1-introduction",
             "search_term": "adjustable-rate mortgage",
-            "search_results_total": 14,
+            "chapter_search_results_total": 14,
+            "rkt_search_results_total": 1,
         },
         "principles-microeconomics-ap-courses-2e": {
             "default_page": "1-introduction",
             "search_term": "positive externality",
-            "search_results_total": 24,
+            "chapter_search_results_total": 24,
+            "rkt_search_results_total": 1,
         },
         "principles-managerial-accounting": {
             "default_page": "1-why-it-matters",
             "search_term": "relevant range",
-            "search_results_total": 51,
+            "chapter_search_results_total": 51,
+            "rkt_search_results_total": 3,
         },
         "principles-management": {
             "default_page": "1-introduction",
             "search_term": "plan is a decision to carry out a particular action",
-            "search_results_total": 11,
+            "chapter_search_results_total": 11,
+            "rkt_search_results_total": 0,
         },
         "psychology-2e": {
             "default_page": "1-introduction",
             "search_term": "event schema",
-            "search_results_total": 15,
+            "chapter_search_results_total": 15,
+            "rkt_search_results_total": 1,
         },
         "statistics": {
             "default_page": "1-introduction",
             "search_term": "memoryless property",
-            "search_results_total": 5,
+            "chapter_search_results_total": 5,
+            "rkt_search_results_total": 1,
         },
         "university-physics-volume-1": {
             "default_page": "1-introduction",
             "search_term": "interference",
-            "search_results_total": 66,
+            "chapter_search_results_total": 66,
+            "rkt_search_results_total": 3,
         },
         "university-physics-volume-2": {
             "default_page": "1-introduction",
             "search_term": "interference fringes",
-            "search_results_total": 3,
+            "chapter_search_results_total": 3,
+            "rkt_search_results_total": 0,
         },
         "university-physics-volume-3": {
             "default_page": "1-introduction",
             "search_term": "interference fringes",
-            "search_results_total": 61,
+            "chapter_search_results_total": 61,
+            "rkt_search_results_total": 1,
         },
         "us-history": {
             "default_page": "1-introduction",
             "search_term": "PATRIOTS",
-            "search_results_total": 20,
+            "chapter_search_results_total": 20,
+            "rkt_search_results_total": 0,
         },
     }
```

Comment:
Might want to move this to another file as it's over 200 lines worth of data at this point...